torch/csrc/jit/python/script_init.cpp

```diff
@@ -471,13 +476,18 @@ static void setInputTensorTypes(Graph& g, const Stack& stack, bool complete) {
     if (auto named_type = v->type()->cast<c10::NamedType>()) {
       if (auto qualname = named_type->name()) {
         if (getCustomClass(qualname->qualifiedName())) {
-          s_iter++;
+          if (param_count_list.empty()) {
+            s_iter++;
+          } else {
+            s_iter += param_count_list[list_idx];
+          }
           continue;
```

Comment:
this continue skips the `list_idx++` below, will this be ok?

Reply:
Good catch. This is a bug and now I fixed it.
It worked because the param_count_list = {1, 3, 3, 1, 1} in `test_small_model`, the two PackeParam has same amount of tensors, so it worked...